README.md

```diff
@@ -28,16 +35,55 @@ This section describes the terms used for all documentation in this repository.
   create an environment to host the container. Neither the guest OS nor the
   guest kernel need to be the same as the host operating system.
 
-  See [the image builder documentation](image-builder/README.md).
+  See the [image creation](#image-creation) section.
 
 - initrd (or "initramfs")
 
-  A compressed cpio archive loaded into memory and used as part of the Linux
-  startup process. During startup, the kernel unpacks it into a special
-  instance of a tmpfs that becomes the initial root file system.
+  A compressed `cpio(1)` archive, created from a rootfs which is loaded into memory and used as part of the Linux startup process. During startup, the kernel unpacks it into a special instance of a `tmpfs` that becomes the initial root filesystem.
 
-  See [the initrd builder documentation](initrd-builder/README.md).
+  See the [initrd creation](#initrd-creation) section.
 
 - "Base OS"
 
-  A particular version of a Linux distribution used to create a Guest OS from.
+  A particular version of a Linux distribution used to create a rootfs from.
+
+## Usage
+
+The top-level `Makefile` contains an example of how to use the available components.
+
+Some components can either run on the host system, or from within a Docker container for ease of setup. The behaviour is controlled by specifying the `USE_DOCKER` variable.
```

Comment:
Hi @klynnrif - please note this new paragraph which needs to be reviewed.

Reply:
Hi @jodh-intel! Only one change here for consistency - I suggest changing "behaviour" to "behavior". I'm not sure we ever settled on whether we will always change to American English, but I know we have been in other docs.

Reply:
Hi @klynnrif - I've reworded to avoid this issue ;)